Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:latest
+WORKDIR /app
+ADD . .
+RUN go mod download
+CMD cd cmd && go run main.go
```

Comment:
Why are not compiling the app before running it, here? What are the disadvantages of `go run`